README.md

```diff
@@ -20,6 +20,12 @@
 
 </br>
 <p> Bailey was here! </p>
+<p> Bailey was here! </p>
+<p> Bailey was here! </p>
+<p> Bailey was here! </p>
+<p> Bailey was here! </p>
+<p> Bailey was here! </p>
+<p> Bailey was here! </p>
```

Comment:
Nice!

Reply:
doies does this send immediately or just as part of the review?

Reply:
fuck